.circleci/config.yml

```
@@ -40,6 +40,17 @@ commands:
                 mentions: maxammann,st,UV99P8ETW
                 only_for_branches: main
                 webhook: ${SLACK_URL}
+    prepare_workspace:
+        description: Attach the workspace at ~/attached_workspace and list its contents
+        steps:
+            - attach_workspace:
+                at: ~/attached_workspace
+            - run:
+                command: ls ~/attached_workspace
+                name: Attached workspace contents
+            - run:
+                command: ls -A ~/attached_workspace
+                name: Recursive attached workspace contents
```

```suggestion
                name: Recursively list attached workspace contents
```

`ls -A` does not work for me to list recursvely the content. You can try to use `tree` though.

---

tree is not installed. ls -R works though

---

Ahh right it just shows hidden files.